test/torchscript_consistency_impl.py

```diff
@@ -83,7 +83,7 @@ def func(_):
             f_max = 20.0
             n_mels = 10
             sample_rate = 16000
-            norm = ""
+            norm = None
```

Comment:
Rather than changing it to `None`, I suggest following [the original suggestion](https://github.com/pytorch/audio/pull/589#discussion_r420324494) of passing string value.